tests/hardware/plugins/decades/test_decades.py

```diff
@@ -38,49 +36,61 @@ def test_close(decades: Decades) -> None:
 
 
 @patch("json.loads")
-@patch("finesse.hardware.plugins.decades.decades.get_decades_data")
+@patch("finesse.hardware.plugins.decades.decades.Decades._get_decades_data")
 def test_on_reply_received_no_error(
-    get_decades_data_mock: Mock, json_loads_mock: Mock, qtbot
+    _get_decades_data_mock: Mock, json_loads_mock: Mock, decades: Decades, qtbot
```

Comment:
I guess you did a "find and replace all"? I think it makes sense to have `_get_decades_data` be private, but I wouldn't put an underscore at the start of the argument names.